docs/documentation/server_admin/topics/organizations/managing-organization.adoc

```diff
@@ -11,6 +11,8 @@ To use organizations, you have to enable the feature for the current realm.
 
 . Toggle *Organizations* to *On*.
 
+. Click *Save*.
```

Comment:
Same thing for this one - should prob be part of a separate PR

Reply:
https://github.com/keycloak/keycloak/pull/33411